example/template.env

```diff
@@ -0,0 +1,29 @@
+# !!! TODO REPLACE WITH ACTUAL RELEASE !!!
+# Image Configuration
+WIKIBASE_IMAGE_NAME=toanwmde/wikibase:1.35.0-wmde.0
+QUERYSERVICE_IMAGE_NAME=toanwmde/wdqs:0.3.40-wmde.0
+QUERYSERVICE_UI_IMAGE_NAME=toanwmde/wdqs-frontend:wmde.0
+ELASTICSEARCH_IMAGE_NAME=toanwmde/elasticsearch:6.5.4-wmde.0
+WIKIBASE_BUNDLE_IMAGE_NAME=toanwmde/wikibase-bundle:1.35.0-wmde.0
+QUICKSTATEMENTS_IMAGE_NAME=toanwmde/quickstatements:wmde.0
+MYSQL_IMAGE_NAME=mariadb:10.3
+
+# Mediawiki Configuration
+MW_ADMIN_NAME=admin
+MW_ADMIN_PASS=password
```

Comment:
This is to short, thus MW fails to install. Setting this to `guestguest` or something like that would work. If we want to provide an intentionally broken value here, this should probably rather be left empty and a comment added.

Reply:
good catch and sorry, my bad didn't try it before changing it back to a default